shared/.semaphore.yml

```diff
@@ -0,0 +1,38 @@
+# Setting for Semaphore CI
+# Copy these configurations to the settings on Semaphore CI
+
+project_settings:
+  env_variables:
+    NODE_ENV: production
```

Comment:
@rosle Setting `NODE_ENV` to production will force `webpacker` to build using production config. 
Should be `test` IMO.

https://github.com/rails/webpacker#custom-rails-environments

Reply:
Also `envs` are defined in `docker-compose` file. Would be better to have all the env configs at one place.

```
- RACK_ENV=test
  RAILS_ENV=test
```

Reply:
@abhinavmsra Not sure about this, I just grab this the config we set in the semaphore project, I'll need to recheck it again. 👍

Reply:
@rosle I think @abhinavmsra is right. Let's change it in this PR and in the projects on Semaphore 🙇